build.fsx

```diff
@@ -191,7 +213,10 @@ Target "Default" DoNothing
 "Default"
   ==> "PrepareRelease"
   ==> "Build.Net35"
+  =?> ("Build.NetCore", isDotnetSDKInstalled)
+  =?> ("RunTests.NetCore", isDotnetSDKInstalled)
   ==> "NuGet"
+  =?> ("Nuget.AddNetCore", isDotnetSDKInstalled)

```

Comment:
Also, I'd be in favor of renaming the "NuGet" target to "NuGet.Paket" so that "NuGet" builds a complete package.